### PR TITLE
[CTCP-4601] add ContainerIdentificationNumber page

### PR DESCRIPTION
**Pull request by prashanthmamidi**

### CODE REVIEW

File: app/views/transportEquipment/index/ContainerIdentificationNumberView.scala.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+button: Button,
+inputText: InputText,
+errorSummary: ErrorSummary
+)
+
+@(form: Form[_], arrivalId: ArrivalId, mrn: MovementReferenceNumber, transportEquipmentIndex: Index, mode: Mode, viewModel: ContainerIdentificationNumberViewModel)(implicit request: Request[_], messages: Messages)
+
+@mainTemplate(
+title = viewModel.title,
+showBackLink = true
```
Comment (JakeGoodwin) [inline app/views/transportEquipment/index/ContainerIdentificationNumberView.scala.html:35]: This page needs formatting with indentation

File: conf/app.transportEquipment.index.routes [hunk 2]
```diff
@@ -4,3 +4,8 @@ GET        /:arrivalId/transport-equipment/:equipmentIndex/items
 POST       /:arrivalId/transport-equipment/:equipmentIndex/items                            controllers.transportEquipment.index.GoodsReferenceController.onSubmit(arrivalId: ArrivalId,  equipmentIndex:Index, mode: Mode = NormalMode)
 GET        /:arrivalId/transport-equipment/:equipmentIndex/change-items                     controllers.transportEquipment.index.GoodsReferenceController.onPageLoad(arrivalId: ArrivalId, equipmentIndex:Index, mode: Mode = CheckMode)
 POST       /:arrivalId/transport-equipment/:equipmentIndex/change-items                     controllers.transportEquipment.index.GoodsReferenceController.onSubmit(arrivalId: ArrivalId,  equipmentIndex:Index, mode: Mode = CheckMode)
+
```
Comment (JakeGoodwin) [inline conf/app.transportEquipment.index.routes:7]: Indentation here too please, you can look at the other routes files for examples. It just makes it nice and easy to read

File: app/controllers/transportEquipment/index/ContainerIdentificationNumberController.scala [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.transportEquipment.index
+
+import controllers.actions._
+import forms.ContainerIdentificationNumberFormProvider
+import models.requests.{DataRequest, MandatoryDataRequest}
```
Comment (JakeGoodwin) [inline app/controllers/transportEquipment/index/ContainerIdentificationNumberController.scala:21]: MandatoryDataRequest import can be removed

File: conf/messages.en [hunk 4]
```diff
@@ -143,6 +143,19 @@ newContainerIdentificationNumber.error.length = The identification number must b
 newContainerIdentificationNumber.error.characters = The identification number must only include letters a to z and numbers 0 to 9
 newContainerIdentificationNumber.error.alreadyEntered = This identification number has already been submitted. Enter a unique container identification number
 
+containerIdentificationNumber.NormalMode.title = What is the container identification number?
+containerIdentificationNumber.CheckMode.title = What is the new container identification number?
+containerIdentificationNumber.NormalMode.heading = What is the container identification number?
+containerIdentificationNumber.CheckMode.heading = What is the new container identification number?
+containerIdentificationNumber.NormalMode.paragraph=
```
Comment (JakeGoodwin) [inline conf/messages.en:150]: empty message key

**Comment by JakeGoodwin** [comment]
It would also be good to have a view spec

File: conf/messages.en (hunk 4 above)
Comment (prashanthmamidi) [reply conf/messages.en:150]: It's on purpose, For Normal mode there's no paragraph in the content pages - also checked with Gavin if this is an expected behaviour? he mentioned it's correct.

**Review by JakeGoodwin — DISMISSED** [review]

**Review by JakeGoodwin — APPROVED** [review]

**Review by JakeGoodwin — APPROVED** [review]

**Pull request merged**